test/logflare/context_cache/cache_buster_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Logflare.ContextCache.CacheBusterTest do
+  use Logflare.DataCase
+
+  alias Cainophile.Changes.DeletedRecord
+  alias Cainophile.Changes.Transaction
+  alias Logflare.ContextCache
+  alias Logflare.ContextCache.CacheBuster
+  alias Logflare.Sources
+
+  setup do
+    insert(:plan, name: "Free")
+    user = insert(:user)
+    source = insert(:source, user: user)
+    [source: source, user: user]
+  end
+
+  test "cache buster", %{source: %{id: source_id}} do
+    for child_spec <- ContextCache.Supervisor.buster_specs() do
+      start_supervised!(child_spec)
+    end
+
+    change = %DeletedRecord{
+      relation: {"public", "sources"},
+      old_record: %{"id" => Integer.to_string(source_id)}
+    }
+
+    test_pid = self()
+
+    Mimic.expect(ContextCache, :bust_keys, fn arg ->
+      send(test_pid, arg)
+    end)
```

Comment:
this does not test the full flow of the key busting.
should set some initial cache keys that are to be busted, then do an expectation that syncing functions are called.